CityAssistant/CityAssistant/Views/CityTourView.swift

```diff
@@ -42,14 +68,14 @@ struct CityTourView: View {
     
     init(pois: [POI]) {
         viewModel = CityTourViewModel(pois: pois)
-        poiViewModel = POIViewModel()
+        poiViewModel = POIViewModel(business: CityTourView_Previews.mockPOI)
```

Comment:
We shouldn't be passing a test object in here. It will be overwritten by whatever business is selected anyways but it doesn't make a ton of sense to do it. I figure that you're probably only doing it because the AR view doesn't have any real POIs at the moment.